.travis.yml

```
@@ -6,44 +6,32 @@ cache:
 
 env:
   global:
-    - COMPOSER_ARGS="--no-interaction"
+    - COMPOSER_ARGS="--no-interaction --ignore-platform-reqs"
     - COVERAGE_DEPS="php-coveralls/php-coveralls"
 
 matrix:
   fast_finish: true
   include:
-    - php: 5.6
+    - php: 7.3
       env:
         - DEPS=lowest
-        - REMOVE_DEV_DEPS="psr/http-factory"
-    - php: 5.6
-      env:
-        - DEPS=latest
-        - REMOVE_DEV_DEPS="psr/http-factory"
-    - php: 7
-      env:
-        - DEPS=lowest
-    - php: 7
-      env:
-        - DEPS=latest
-    - php: 7.1
-      env:
-        - DEPS=lowest
-    - php: 7.1
+    - php: 7.3
       env:
         - DEPS=latest
         - CS_CHECK=true
         - TEST_COVERAGE=true
-    - php: 7.2
+    - php: 7.4
       env:
         - DEPS=lowest
-    - php: 7.2
+    - php: 7.4
+      env:
+        - DEPS=latest
+    - php: 'nightly'
       env:
         - DEPS=latest
```

Same as in all other components:
```suggestion
    - php: nightly
      env:
        - DEPS=lowest
    - php: nightly
      env:
        - DEPS=latest
```